src/config/datatables.php

```diff
@@ -39,19 +39,21 @@
      * This is where you can register your custom dataTables builder.
      */
     'engines'        => [
-        'eloquent'   => \Yajra\DataTables\EloquentDataTable::class,
-        'query'      => \Yajra\DataTables\QueryDataTable::class,
```

Comment:
I think we can retain `query` engine to prevent breaking change. This way, users with old config won't break?

Also, this would also become an additional feature where we can just use `DataTables::query()` for a shorter syntax?

Reply:
I added a Deprecated test as well to make sure we still support the old config's. Maybe I missed a use-case in that? See: https://github.com/pimlie/laravel-datatables/blob/feature-support-plugin-engine-methods/tests/Integration/DeprecatedQueryEngineTest.php

Reply:
Yeah my mistake, I think this is not a breaking change but I think it's nice to have `query()` engine? Thanks!